connectors/kafka-connector/src/main/scala/zio/connect/kafka/LiveKafkaConnector.scala

```diff
@@ -0,0 +1,58 @@
+package zio.connect.kafka
+
+import org.apache.kafka.clients.producer.ProducerRecord
+import zio._
+import zio.connect.kafka.KafkaConnector.{NewTopic, TopicListing}
+import zio.kafka.admin.AdminClient.{NewTopic => AdminNewTopic}
+import zio.kafka.admin._
+import zio.kafka.consumer._
+import zio.kafka.producer._
+import zio.kafka.serde._
+import zio.stream._
+
+case class LiveKafkaConnector(adminClient: AdminClient, consumer: Consumer, producer: Producer) extends KafkaConnector {

```

Comment:
We also need a TestKafkaConnector so we can give users the option of using an inMemory version of KafkaConnector.

Reply:
Do we really need this? Because with kafka you can already use EmbeddedKafka for in memory version.

Reply:
That would not give them a KafkaConnector test implementation.